Packs/ThreatConnect/Integrations/ThreatConnect_v3/ThreatConnect_v3.py

```diff
@@ -0,0 +1,1646 @@
+        })
+
+    context = {
+        'TC.Group.Indicator(val.GroupID && val.GroupID === obj.GroupID && val.IndicatorID && val.IndicatorID === '
+        'obj.IndicatorID)': contents
+    }
+
+    return_outputs(
+        tableToMarkdown('ThreatConnect Group Indicators', contents, removeNull=True),
+        context,
+        response
+    )
+
+
+def get_group_attributes(client: Client) -> None:  # pragma: no cover
+    group_id = demisto.args().get('group_id')
+    response, status_code = list_groups(client, return_raw=True, include_attributes='true', group_id=group_id)
+    if status_code != 200:
+        return_error('Error from the API: ' + response.get('message',
+                                                           'An error has occurred if it persist please contact your '
+                                                           'local help desk'))
+    attributes = response.get('data')[0].get('attributes', {}).get('data', [])
+    contents = []
+    headers = ['AttributeID', 'Type', 'Value', 'DateAdded', 'LastModified', 'Displayed']
+    for attribute in attributes:
+        contents.append({
+            'GroupID': group_id,
+            'AttributeID': attribute.get('id'),
+            'Type': attribute.get('type'),
+            'Value': attribute.get('value'),
+            'DateAdded': attribute.get('dateAdded'),
+            'LastModified': attribute.get('lastModified'),
+            'Displayed': attribute.get('displayed')
+        })
+
+    context = {
+        'TC.Group.Attribute(val.GroupID && val.GroupID === obj.GroupID && val.AttributeID && val.AttributeID ==='
+        ' obj.AttributeID)': contents
+    }
+
+    return_outputs(
+        tableToMarkdown('ThreatConnect Group Attributes', contents, headers, removeNull=True),
+        context,
+        response
+    )
+
+
+def get_group_security_labels(client: Client) -> None:  # pragma: no cover
+    group_id = demisto.args().get('group_id')
+    response, status_code = list_groups(client, return_raw=True, include_security_labels='true', group_id=group_id)
+    if status_code != 200:
+        return_error('Error from the API: ' + response.get('message',
+                                                           'An error has occurred if it persist please contact your '
+                                                           'local help desk'))
+    security_labels = response.get('data')[0].get('securityLabels', {}).get('data', [])
+    contents = []
+    headers = ['Name', 'Description', 'DateAdded']
+    for security_label in security_labels:
+        contents.append({
+            'GroupID': group_id,
+            'Name': security_label.get('name'),
+            'Description': security_label.get('description'),
+            'DateAdded': security_label.get('dateAdded')
+        })
+
+    context = {
+        'TC.Group.SecurityLabel(val.GroupID && val.GroupID === obj.GroupID && val.Name && val.Name === '
+        'obj.Name)': contents
+    }
+
+    return_outputs(
```

Comment:
use `CommandResults` and `return_results` instead of `return_outputs` and `demisto.results`